### PR TITLE
Add new sample with the implementation of the batch operations in node js

### PR DESCRIPTION
**Pull request by JhontSouth**
## Description
This PR adds the new sample _**bot-batch-operations**_ that includes the uses of the Teams batch APIs in node js.

## Specific Changes
- Created the _**bot-batch-operations**_ sample in nodejs.
- Added the logic to use the new features in _**TeamsBatchOperationsBot.**_
- Added the README guide to use the new bot.

## Testing
The following images show some sample bot responses.
GetFailedEntries
MessageAListOfUsers

### CODE REVIEW

File: samples/bot-batch-operations/nodejs/assets/sample.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+[
```
Comment (ceciliaavila) [inline samples/bot-batch-operations/nodejs/assets/sample.json:1]: This file should be updated with the sample name and capacities.

File: samples/bot-batch-operations/nodejs/bots/teamsConversationBot.js [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+const {
+    ActionTypes,
+    CardFactory,
+    MessageFactory,
+    TeamsActivityHandler,
+    TeamsInfo,
+    TurnContext
+} = require('botbuilder');
+
+class TeamsConversationBot extends TeamsActivityHandler {
```
Comment (ceciliaavila) [inline samples/bot-batch-operations/nodejs/bots/teamsConversationBot.js:13]: This should be called TeamsBatchOperationsBot

File: samples/bot-batch-operations/nodejs/bots/teamsConversationBot.js [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+            await next();
+        });
+
+        // This method registers the lambda function, which will be invoked when message sent by user is updated in chat.
+        this.onTeamsMessageEditEvent(async (context, next) => {
+            let editedMessage = context.activity.text;
+            await context.sendActivity(`The edited message is ${editedMessage}"`);
+            next();
+        });
+    }
```
Comment (ceciliaavila) [inline samples/bot-batch-operations/nodejs/bots/teamsConversationBot.js:65]: This method is not used in this bot. Please remove unused functionality.

File: samples/bot-batch-operations/nodejs/index.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// index.js is used to setup and configure your bot
+
+// Import required pckages
```
Comment (ceciliaavila) [inline samples/bot-batch-operations/nodejs/index.js:6]:
```suggestion
// Import required packages
```

File: samples/bot-batch-operations/nodejs/index.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+const restify = require('restify');
+
+// Import required bot services.
+// See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const {
+    CloudAdapter,
+    ConfigurationBotFrameworkAuthentication
+} = require('botbuilder');
+
+const { TeamsConversationBot } = require('./bots/teamsConversationBot');
```
Comment (ceciliaavila) [inline samples/bot-batch-operations/nodejs/index.js:22]: The class name and the file name should be changed.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

File: samples/bot-batch-operations/nodejs/assets/sample.json [hunk 6]
```diff
@@ -35,8 +35,8 @@
       {
         "type": "Image",
         "order": 100,
-        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/bot-conversation/nodejs/Images/ConversationBot.gif",
-        "alt": "Solution UX showing conversation events for teams bot"
+        "url": "https://raw.githubusercontent.com/southworks/Microsoft-Teams-Samples/bf92bc9c53ed366037c70c51af8432782575cecf/samples/bot-batch-operations/nodejs/Images/GetFailedEntries.png",
```
Comment (ceciliaavila) [inline samples/bot-batch-operations/nodejs/assets/sample.json:38]: Don't use the SW fork. Leave the main branch URL even when the sample does not exist yet.

File: samples/bot-batch-operations/nodejs/assets/sample.json [hunk 7]
```diff
@@ -35,8 +35,8 @@
...
+        "alt": "Image showing one of the Batch Operations (Get Failed Entres) available from Bot Framework v4"
```
Comment (ceciliaavila) [inline samples/bot-batch-operations/nodejs/assets/sample.json:39]: Be careful with the typos.
```suggestion
        "alt": "Image showing one of the Batch Operations (Get Failed Entries) available from Bot Framework v4"
```

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Review by ceciliaavila — APPROVED** [review]

**Review by sw-joelmut — APPROVED** [review]

**Pull request closed without merging**